src/styles.css

```diff
@@ -78,3 +78,45 @@
 .location-submit-btn {
   width: 21.75rem;
 }
+
+.location-card {
+  max-width: 850px;
+}
+
+.location-radio-group {
+  margin: 0 auto;
+  background-color: var(--omrs-color-bg-high-contrast);
+  border: 2px solid rgba(249, 249, 249, 0.94);
+  list-style: none;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+}
+
+.location-radio-group input[type="radio"] {
+  opacity: 0;
+  position: fixed;
+  width: 0;
+}
+
+.location-radio-group label {
+  display: flex;
+  background-color: var(--omrs-color-bg-high-contrast);
+  color: var(--omrs-color-primary-blue);
+  padding: 5px;
+  font-family: sans-serif, Arial;
```

Comment:
The font family of choice in the styleguide is Roboto and in a few cases Work Sans. There are some typography styles defined in the styleguide as well thar you might find useful, take a look,  https://styleguide.openmrs.org

Reply:
Thanks - will do.  This snuck in, I didn't mean to set the font here.

Reply:
Removed these font-family and font-size settings, in favor of defaults